canonical/maas-ui
# refactor: merge sidepanel types MAASENG-1857

## Done

- refactor: merge sidepanel types MAASENG-1857
  - rename `getHeaderTitle` to `getSidePanelTitle`

## QA

### QA steps

- Go to machine list
- Open a side panel and verify it has a correct title
- Go to AZ
- Open a side panel and verify it has a correct title
- Go to any other page containing a side panel and verify it has a correct title

## Fixes

https://warthogs.atlassian.net/browse/MAASENG-1857
